app.js

```diff
@@ -253,25 +247,37 @@ client.on("message", (msg) => {
       }
     }
   }
+  if (msg.content === "get help" || msg.content === "get article") {
+    msg.reply("```Are you trying to call me?```")
+  }
 
   if (msg.content.startsWith(prefix + "set article ")) {
     correctTimeProvided = false;
+    displayTime = true;
```

Comment:
What is this `displayTime`?

Reply:
It's a boolean variable which will not display time in case of any error  in set time and get time commands

Reply:
We aren't displaying the time anywhere right, better change the variable name

Reply:
okk i will do it